### PR TITLE
Task1 part1 layout structuring and slicing

### PR DESCRIPTION
**Pull request by ashrafabuodeh**

### CODE REVIEW

**Review by abood454 — APPROVED** [review]
great work

File: task_1/index.html [hunk 1]
```diff
@@ -4,10 +4,48 @@
...
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+            <div class="coffee-item"></div>
+        </div>
+
+    </div>
+    
+
+
```
Comment (araraltawarah98) [inline task_1/index.html]: Remove spaces

**Review by araraltawarah98 — COMMENTED** [review]
nice work

**Review by araraltawarah98 — APPROVED** [review]

**Review by laithyounes-archive — CHANGES_REQUESTED** [review]
please follow @araraltawarah98 comments, and add a newline at the end of the file!
other than that LGTM

**Review by Mohsen-Awad — APPROVED** [review]
Good work

**Review by Mohsen-Awad — COMMENTED** [review]
Good work

**Review by laithyounes-archive — APPROVED** [review]

**Review by abood454 — APPROVED** [review]

**Review by Mohsen-Awad — APPROVED** [review]

**Review by araraltawarah98 — APPROVED** [review]

**Pull request merged**